CHANGELOG.md

```diff
@@ -22,5 +32,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Missing profile language in approval view
 - Photo usage approval not working when set from approval view
 - Missing Helsinki profile link in mobile menu
+- Non existent dates being silently transformed into existing dates
```

Comment:
This fix is implemented with version `1.1.0`. Looks like it has been added into the wrong place in the changelog in commit https://github.com/City-of-Helsinki/youth-membership-ui/pull/171/commits/8e10356f6bb4731815fbce7eb202e1499854e3d5

Reply:
I moved it to 1.1.0 fixes.